TOOLS/mpv_identify.sh

```diff
@@ -79,7 +79,6 @@ EOF
         video-bitrate
         video-codec
         video-format
-        video-aspect
```

Comment:
This should be `video-params/aspect` or maybe `video-params/aspect-name` (whatever you prefer). `video-aspect` doesn't even currently work.

Reply:
Good point, I was just removing references to this prop. Replaced.